### PR TITLE
Fix new message list

### PR DESCRIPTION
**Pull request by ogkuzmin**
### Description

Fix max width decorator logic

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml [hunk 1]
```diff
@@ -21,6 +21,9 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/header"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml:25]: NIT: Those two margins can be replaced with `layout_marginHorizontal`

**Review by adasiewiczr — DISMISSED** [review]

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/MaxPossibleWidthDecorator.kt [hunk 2]
```diff
@@ -11,19 +11,23 @@ internal class MaxPossibleWidthDecorator : BaseDecorator() {
         viewHolder: OnlyMediaAttachmentsViewHolder,
         data: MessageListItem.MessageItem
     ) {
-        setMaxWidthToView(viewHolder.binding.mediaAttachmentsGroupView)
+        (viewHolder.binding.mediaAttachmentsGroupView.layoutParams as ConstraintLayout.LayoutParams).apply {
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/MaxPossibleWidthDecorator.kt:14]: Nit: could use `updateLayoutParams`

**Review by zsmb13 — APPROVED** [review]

File: stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml (hunk 1 above)
Comment (ogkuzmin) [reply stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml]: Thanks

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/MaxPossibleWidthDecorator.kt (hunk 2 above)
Comment (ogkuzmin) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/MaxPossibleWidthDecorator.kt]: Thanks applied here and in other place

**Review by zsmb13 — APPROVED** [review]

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**